api/envoy/config/route/v3/route_components.proto

```diff
@@ -833,6 +833,15 @@ message RouteAction {
       type.matcher.v3.RegexMatchAndSubstitute regex_rewrite = 2;
     }
 
+    // CookieAttribute defines an API for adding additional attributes for a HTTP cookie.
+    message CookieAttribute {
+      // The name of the cookie attribute.
+      string name = 1 [(validate.rules).string = {min_len: 1}];
+
+      // The optional value of the cookie attribute.
+      string value = 2;
```

Comment:
Should the characters allowed in here be limited by PGV and/or config checking?

cc @RyanTheOptimist @yanavlasov

Reply:
Good point. Not sure what is the max length we can go up to. Would be open to ideas.

Reply:
It's not just the length, the actual characters as specified by the HTTP spec. We limit header names, etc. for similar reasons.